roofit/roofitcore/src/RooAbsPdf.cxx

```diff
@@ -976,13 +979,55 @@ RooAbsReal* RooAbsPdf::createNLL(RooAbsData& data, const RooLinkedList& cmdList)
   pc.defineDouble("IntegrateBins", "IntegrateBins", 0, -1.);
   pc.defineMutex("Range","RangeWithName") ;
   pc.defineMutex("GlobalObservables","GlobalObservablesTag") ;
+  pc.defineInt("NewStyle", "NewStyle", 0, 0);
```

Comment:
Since you are adding new configuration arguments, you also have to explain them in the doxygen code in the same C++ file, such that they appear in the docs to `createNLL` and [`fitTo`](https://root.cern.ch/doc/master/classRooAbsPdf.html#a5f79f16f4a26a19c9e66fb5c080f59c5).